ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/Errors.xml

```diff
@@ -148,6 +148,7 @@
     <Message value="{0}">
       <Argument>getLambdaCanBeReplacedWarningText replaceCandidate</Argument>
     </Message>
+    <QuickFix>ReplaceLambdaFix</QuickFix>
```

Comment:
Let's try using a clearer name? Replace lambda with what?

Reply:
Maybe 'with expression'?

P.S. 
to replace it with 'id', I plan to write a separate quickfix